CHANGES.rst

```diff
@@ -1,7 +1,8 @@
 0.8.4 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Automatically detect the appropriate ``host_owner`` if undefined, in
+  ``check-branches``.
```

Comment:
If `check-branches` breaks if the configuration defines `host_owner`, we should perhaps mention the breaking change. Suggestion:

```
* |backward-incompatible| **check-branches**: Drop ``host_owner`` option in the configuration file. You should remove this option, otherwise an error will occur while reading the configuration file
```

And add the following at the end of the file:

```
.. role:: raw-html(raw)
.. |backward-incompatible| raw:: html

    <span style="background-color: #ffffbc; padding: 0.3em; font-weight: bold;">backward incompatible</span>
```

I have used that in the [changelog of django-cid](https://django-correlation-id.readthedocs.io/en/latest/history.html#id1) (and elsewhere) to highlight breaking changes.

Reply:
Good idea!